.circleci/config.yml

```diff
@@ -170,6 +170,71 @@ jobs:
       - store_artifacts:
           path: monitor-ci/test-artifacts/results/remocal
           destination: test_artifacts/results/remocal
+
+  oss-changelog:
+    docker:
+      - image: quay.io/influxdb/changelogger:0dbf37ba8a912460358d322dcad414b80ce074f2
```

Comment:
I'm pinning it at that tag specifically in an attempt to prevent any breaking changes in the `changelogger` image from breaking this process unknowingly. I think the behavior of the image should be pretty well fixed at this point so there shouldn't be a need to frequently update this tag here.